indico/modules/events/editing/client/js/components/EditingTagManagement.jsx

```diff
@@ -172,3 +178,14 @@ export default function TagManager({eventId}) {
 TagManager.propTypes = {
   eventId: PropTypes.number.isRequired,
 };
+
+export default function EditingTagManagement() {
```

Comment:
Why not just editing TagManager and adding
```
const eventId = useNumericParam('confId');
...
<ManagementPageBackButton url={dashboardURL({confId: eventId})} />
<ManagementPageSubTitle title={Translate.string('Tags')} />
...
```

Reply:
The problem of doing this in `TagManager` is that we return `null` or a `Loader` in some cases - but we always need to render those two components to have the correct title/subtitle on the page. So having a separate component for it is easier.

Also, it separates the actual tag manager from setting up the surrounding page.